var/spack/repos/builtin/packages/libmypaint/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Libmypaint(AutotoolsPackage):
+    """libmypaint - MyPaint brush engine library
```

Comment:
```suggestion
    """libmypaint - MyPaint brush engine library.
```
When you run `spack info libmypaint`, Spack removes all indentation/newlines, so periods help break apart sentences

Reply:
Will do!